client/landing/jetpack-cloud/sections/scan/upsell.jsx

```diff
@@ -18,6 +18,18 @@ import Upsell from 'landing/jetpack-cloud/components/upsell';
 import { getSelectedSiteSlug } from 'state/ui/selectors';
 import { recordTracksEvent } from 'state/analytics/actions';
 
+function ScanMultisiteBody() {
+	const translate = useTranslate();
+	return (
+		<Upsell
+			headerText={ translate( 'Your site does not have scan' ) }
+			bodyText={ translate( 'Scan is currently not supported on WordPress Multi-site setups.' ) }
```

Comment:
```suggestion
			bodyText={ translate( 'Scan is currently not supported on WordPress multi-site networks.' ) }
```

Reply:
We could also consider linking multi-site networks to support about those sites.